boards/waveshare/esp32s3_matrix/esp32s3_matrix_esp32s3_procpu_defconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024 Joel Guittet
+# Copyright (c) 2025 Chen Xingyu <hi@xingrz.me>
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_GPIO=y
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_INPUT=y
+CONFIG_LED_STRIP=y
```

Comment:
no main stack size, no input, no led strip

Reply:
Since it's a driver within a driver of another class, an overlay for the sample is fine